vuln/core/README.txt

```diff
@@ -0,0 +1,631 @@
+description: Truncated packet could crash via OOB read
+vulnerable: 4.x, 5.x, 6.x, 7.x
+patched: ^4.7.3, ^6.9.5, ^7.5.0
+ref: https://nodejs.org/en/blog/vulnerability/openssl-january-2017/
+overview:
+This is a moderate severity flaw in OpenSSL. By default, Node.js disables RC4 so
+most users are not affected. As RC4 can be enabled programmatically, it is
+possible for a Node.js developer to craft code that may be vulnerable to this
+flaw. Any user activating RC4 in their codebase should prioritise this update.
+
+All active versions of Node.js are affected, but the severity is very low for
+most users.
+
+--
+
+cve: CVE-2017-3732
+vulnerable: 4.x, 5.x, 6.x, 7.x
+patched: ^4.7.3, ^6.9.5, ^7.5.0
+ref: https://nodejs.org/en/blog/vulnerability/openssl-january-2017/
+description: BN_mod_exp may produce incorrect results on x86_64
+overview:
+As noted by the OpenSSL team, the likelihood of being able to craft a practical
+attack that uses this flaw is very low. In addition, Node.js enables
+SSL_OP_SINGLE_DH_USE, further decreasing the chance of a successful exploit of
+this vulnerability in a Node.js service.
+
+All active versions of Node.js are affected, but the severity is very low for
+Node.js users.
+
+--
+
+cve: CVE-2016-7055
+description: Montgomery multiplication may produce incorrect results
+vulnerable: 4.x, 5.x, 6.x, 7.x
+patched: ^4.7.3, ^6.9.5, ^7.5.0
+ref: https://nodejs.org/en/blog/vulnerability/openssl-january-2017/
+overview:
+Some calculations, when run on an Intel Broadwell or later CPU, can produce in
+erroneous results. This flaw has been previously discussed by the Node.js team
+on GitHub. It is not believed that practical attacks can be crafted to exploit
+this vulnerability except in very specific circumstances. Therefore this is a
+low severity flaw.
+
+All active versions of Node.js are affected, but the severity is very low for
+Node.js users.
+
+--
+
+description: potential buffer overflow when writing data to console on Windows 10
+vulnerable: ^7.1.0
+patched: ^7.2.0
+cve: CVE-2016-9551
+ref: https://groups.google.com/forum/#!topic/nodejs-sec/-Tns2vdD-84
+ref: https://github.com/nodejs/node/pull/9647
+overview:
+Please be aware that Node.js v7.2.0 was released today and includes a small
+security update arising from libuv: https://nodejs.org/en/blog/release/v7.2.0/
+
+libuv v1.10.1 reverts a change that was introduced in v1.10.0, included in
+Node.js v7.1.0. The reverted code was found to contain a potential buffer
+overflow in output written to the console. We are not aware of any exploit of
+this flaw and it only impacts Windows 10 (November update and later). This flaw
+has been assigned the identifier CVE-2016-9551 and was originally discovered and
+reported by Hitesh Kanwathirtha of Microsoft.
+
+Users of the v7 release line running on Windows 10 should upgrade to Node.js
+v7.2.0 at their earliest convenience.
+
+No other version of Node.js is known to be impacted by this flaw.
+
+--
+
+XXX break into 4 records?
```

Comment:
is breaking this into a record per CVE busy work, or does it make the data much easier to consume? I suspect the latter, I'll probably make sure that no record has more than a single CVE, that seems to be what rubysec does, but perhaps I misundertand the CVE purposes. Comments?